libr/core/panels.c

```diff
@@ -5494,7 +5501,7 @@ void r_save_panels_layout(RCore *core, const char *_name) {
 		pj_kn (pj, "h", panel->view->pos.h);
 		pj_end (pj);
 	}
-	FILE *file = fopen (config_path, "w");
+	FILE *file = fopen (config_path, "a");
```

Comment:
Use sandbox apis. This is a security flaw. Also, better specify “b” to avoid windows/unix newline issues

Reply:
also, if you save multiple times the file is always growing and it just becomes unhandlable

Reply:
but  i understand the "bug" you are reporting here.. which implies not being able to save more than one layout. merging, but this needs a cleaner solution, otherwise autosaving layout results in HELL